htmresearch/frameworks/pytorch/sparse_speech_experiment.py

```diff
@@ -177,9 +217,13 @@ def createLearningRateScheduler(self, params, optimizer):
     if lr_scheduler is None:
       return None
 
-    lr_scheduler_params = params.get("lr_scheduler_params", None)
```

Comment:
I guess i need to rethink the way we pass parameters to the `lr_scheduler`. This logic should be handled by the `expsuite`. I will take a look and see if i can fix it there

Reply:
Thanks.